### ISSUE TITLE
Allow setting Telemetry Output Logs to Trace always and independently

### ISSUE THREAD

**Issue by sbatten** [microsoft/vscode#227010]
Type: <b>Feature Request</b>

I'd like to set `Telemetry` and `Extension Telemetry` to log level trace so they are always visible for me, but I don't want to have all of VS Code set to trace all the time. The quick pick for managing these log levels doesn't include them at present.

VS Code version: Code - Insiders 1.93.0-insider (d1388fd24fc0acf17ae1f759e85c1acf559ed759, 2024-08-28T05:14:36.262Z)
OS version: Windows_NT x64 10.0.22631
Modes:

**Comment by lramos15** [comment]
@sandy081 Let me know how I could opt into this feature

**Comment by sandy081** [comment]
At present it is explicitly disabled to set log level for telemetry 

https://github.com/microsoft/vscode/blob/c95571667318a8ff30e27e7302f29cc6318a4001/src/vs/workbench/contrib/logs/common/logsActions.ts#L99-L101

This is added for a reason. I would recommend to come up with a new command say `Developer: Show Telemetry Log` and `Developer: Hide Telemetry Log` and toggle log level of Telemetry Logs.

**Comment by lramos15** [comment]
Why is this hidden for the telemetry output channel? The feature request is about setting the level to trace and have it persist so that you can always see the output channel

**Comment by sandy081** [comment]
Please note that we log telemetry events only when the log level is trace, hence it does not makes sense to configure other log levels for  telemetry output and therefore it is hidden. 

IMO this is an implementation detail. As an user, I would need following features or commands

- Developer: Show Telemetry (Session)
- Developer: Show Telemetry (Always)
- Developer: Hide Telemetry

Internally, you can toggle log level for telemetry logger to `trace` for session or always depending upon the command that is called.

**Comment by sbatten** [comment]
@sandy081 do we have other examples of output channels working outside of the system like this?

**Comment by sandy081** [comment]
yeah, network logs. I would be adopting the same strategy for them too.
